virttest/utils_test/qemu.py

```diff
@@ -148,46 +149,57 @@ def check_driver_verifier(session, driver, timeout=300):
 
 
 @error_context.context_aware
-def setup_win_driver_verifier(session, driver, vm, timeout=300):
+def setup_win_driver_verifier(driver, vm, timeout=300):
     """
     Enable driver verifier for windows guest.
 
-    :param session: VM session.
     :param driver: The driver which needs enable the verifier.
     :param vm: VM object.
     :param timeout: Timeout in seconds.
     """
-    verifier_status = check_driver_verifier(session, driver)[1]
-    if not verifier_status:
-        error_context.context("Enable %s driver verifier" % driver,
-                              logging.info)
-        verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
-        session.cmd(verifier_setup_cmd,
-                    timeout=timeout,
-                    ignore_all_errors=True)
-        session = vm.reboot(session)
-        verifier_status, output = check_driver_verifier(session, driver)
+    try:
+        session = vm.wait_for_login(timeout=timeout)
```

Comment:
If it's true that the session may be changed during test, I think the PR can be accepted.
If not, It's really unnecessary to login seperately